pages/questions/questions.css

```diff
@@ -0,0 +1,95 @@
+@import "../../css/dafault.css";
+@import "../../css/navbar.css";
+
+.section-wrapper {
+  min-height: 80vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 8rem;
+}
+
+.questions-card-container .card.questions-card {
+  max-width: 450px;
+  width: 100%;
+  background: var(--secondary-color);
+  border: none;
+}
+
+.questions-card-container .card-body {
+  color: var(--white);
+  padding: 1rem;
+}
+
+.questions-card-container .card-body .card-title {
+  margin: 0.5rem 0 2rem 0;
+  font-size: 1.5rem;
+  text-align: center;
+}
+
+.questions-card-container .card-body .top-display {
+  display: flex;
+  justify-content: space-between;
+}
+
+.questions-card-container .card-body .top-display .count .text {
```

Comment:
we can give a class Name to the element. It might create specificity issues that is had to figure out

Reply:
Yes, these all are separate class names given to the nested elements

Reply:
cool!